### ISSUE TITLE
muted players can still msg

### ISSUE THREAD

**Issue by mrcoffee1026** [kangarko/ChatControl-Pro#851]
Just found that out.  Yup.  Thanks.

**Comment by mrcoffee1026** [comment]
Disabled_Commands: 
  - /mail
  - /me
  - /minecraft:me
  - /tell
  - /msg
  - /r
  - /w
  - /tp
  - /tpa
  - /tpahere
  - /memo
  - /f
  - /kittycannon
  - /describe
  - /eme
  - /etell

**Comment by kangarko** [comment]
Are we talking about individually muted players or /chatcontrol mute command? We do not have individual mute function.

**Comment by mrcoffee1026** [comment]
Yes muted using /mute from Essentials which chatcontrol is now paying attention to.  It's not effective if chatcontrol does not also pay attention to the setting.
